src/Tribe/Integrations/WPML/Filters.php

```diff
@@ -65,8 +66,9 @@ public function filter_tribe_events_rewrite_i18n_slugs_raw( $bases, $method, $do
 		 */
 		$untranslated_bases = array_combine( array_keys( $bases ), array_column( $bases, 0 ) );
 
+		$flags = I18n::COMPILE_STRTOLOWER;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 12 spaces but found 1 space

<sub>hash: 612f9c819b5e78be711451ddc3fe7f2c</sub>